internal/kafka/internal/workers/clusters_mgr.go

```diff
@@ -501,6 +506,42 @@ func (c *ClusterManager) reconcileClusterInstanceType(cluster api.Cluster) error
 	return nil
 }
 
+func (c *ClusterManager) reconcileDynamicCapacityInfo(cluster api.Cluster) error {
+	updatedDynamicCapacityInfo := map[string]api.DynamicCapacityInfo{}
+
+	if c.DataplaneClusterConfig.IsDataPlaneAutoScalingEnabled() {
+		previousDynamicCapacityInfo, err := cluster.RetrieveDynamicCapacityInfo()
+		if err != nil {
+			return err
+		}
+
+		if len(previousDynamicCapacityInfo) > 0 {
+			return nil
+		}
```

Comment:
Yes. We want to basically say that storing cluster capacity info is a one time configuration and only happens when the machine pools have been created. We are not reconciling it each time since we are not at the moment making sure that the machine pools are reconciled at OCM level if the config changes etc

This ready clusters reconciliation is only added to allow the migration of manual clusters (in the database) which will have the machine pool created manually.

Reply:
Thanks for clarifying this! :) It's also answered my question from another comment. Resolving both.